src/client/websocket_client.ts

```diff
@@ -13,12 +14,12 @@ class WebSocketClient {
     token: string|null;
     sequence: number;
     connectFailCount: number;
-    eventCallback?: Function;
-    firstConnectCallback?: Function;
-    reconnectCallback?: Function;
-    errorCallback?: Function;
-    closeCallback?: Function;
-    connectingCallback?: Function;
+    eventCallback?: (message: any) => void;
```

Comment:
The new version of typescript-eslint doesn't want us to use Function any more, so I had to properly define what types of functions we were looking for.